grafana-dashboards/Kafka-Consumer-Metrics.json

```diff
@@ -73,7 +111,7 @@
       "collapsed": false,
       "datasource": {
         "type": "prometheus",
-        "uid": "${DS_PROMETHEUS}"
+        "uid": "PBFA97CFB590B2093"
```

Comment:
We need to retain the DS_PROMETHEUS references, otherwise it doesn't connect up to a prometheus where the UID for the data source is different. 
Not sure why this change has only happened on this one.